tests/mechanic/team_test.py

```diff
@@ -48,17 +48,17 @@ def test_load_known_car(self):
         car = team.load_car(self.team_dir, ["default"], car_params={"data_paths": ["/mnt/disk0", "/mnt/disk1"]})
         assert car.name == "default"
         assert car.config_paths == [os.path.join(current_dir, "data", "cars", "v1", "vanilla", "templates")]
-        assert car.root_path is None
+        assert car.root_path == []
```

Comment:
Nit: When looking at this change it struck me, `root_path` could be renamed to `root_paths` in `Car` class.

Reply:
So, I attempted this just now again. And the issue is we equate `Car` and `Plugin` in several places, just referring to `component.root_path` directly. 

So, instead of forcibly rewriting those to conditionally use the correct field, I will just keep it as is for now.

Reply:
Ok, thanks for checking.